localization/ndt_scan_matcher/include/ndt_scan_matcher/ndt_scan_matcher_core.hpp

```diff
@@ -107,40 +116,61 @@ class NDTScanMatcher : public rclcpp::Node
   NDTScanMatcher();
 
 private:
-  void serviceNDTAlign(
+  void service_ndt_align(
     const tier4_localization_msgs::srv::PoseWithCovarianceStamped::Request::SharedPtr req,
     tier4_localization_msgs::srv::PoseWithCovarianceStamped::Response::SharedPtr res);
 
-  void callbackMapPoints(sensor_msgs::msg::PointCloud2::ConstSharedPtr pointcloud2_msg_ptr);
-  void callbackSensorPoints(sensor_msgs::msg::PointCloud2::ConstSharedPtr pointcloud2_msg_ptr);
-  void callbackInitialPose(
+  void callback_map_points(sensor_msgs::msg::PointCloud2::ConstSharedPtr pointcloud2_msg_ptr);
+  void callback_sensor_points(sensor_msgs::msg::PointCloud2::ConstSharedPtr pointcloud2_msg_ptr);
+  void callback_initial_pose(
     geometry_msgs::msg::PoseWithCovarianceStamped::ConstSharedPtr pose_conv_msg_ptr);
-  void callbackRegularizationPose(
+  void callback_regularization_pose(
     geometry_msgs::msg::PoseWithCovarianceStamped::ConstSharedPtr pose_conv_msg_ptr);
 
-  geometry_msgs::msg::PoseWithCovarianceStamped alignUsingMonteCarlo(
+  NdtResult align(const geometry_msgs::msg::Pose & initial_pose_cov_msg);
```

Comment:
The variable name does not match the type name.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1757/commits/b255b43cde6c73522ef6cd4a66bd2af58cdffedb